docs/src/domains/steps/dns.md

```diff
@@ -1,170 +1,111 @@
-When you come to configuring your DNS provider, replace the suggested `CNAME` record with the custom record pointing from your domain to the target.
-
-Examples of such workaround records and providers include:
-
-<!-- vale Platform.condescending = NO -->
-* `CNAME` flattening at [CloudFlare](https://developers.cloudflare.com/dns/additional-options/cname-flattening)
-* `ANAME` records at [easyDNS](https://easydns.com/features/aname-root-domain-alias/),
-  [DNS Made Easy](https://support.dnsmadeeasy.com/support/solutions/articles/47001001412-aname-records),
-  and [Name.com](https://www.name.com/support/articles/115010493967-adding-an-aname-alias-record)
-* `ALIAS` records at [DNSimple](https://support.dnsimple.com/articles/alias-record/)
-  and [ClouDNS](https://www.cloudns.net/wiki/article/18/)
-<!-- vale Platform.condescending = YES -->
-
-<--->
-
-+++
-title=Use domain forwarding
-file=none
-highlight=false
-+++
-
-If your registrar doesn't support custom records, you can consider using domain forwarding.
-
+To map incoming requests to the appropriate container,
+Platform.sh runs routers in [each region](../../development/regions.md).
+A router's IP address can change in two cases:
+- During an upgrade or maintenance operation, routers can be taken offline while changes are applied.
+- During a region upscale or downscale, routers can be added or removed.
+ 
+In such situations, using `A` records can cause downtime.
+`A` records map a domain to an IP address.
+When a router's IP address changes,
+you need to manually update your `A` records.
+Your site remains offline until you do,
+or until the router is back from maintenance.
+ 
+To prevent downtime, create `CNAME` records through your DNS provider.
+`CNAME` records map a domain to another.
+Set up your `CNAME` records to point at your project's [target URL](../../domains/steps/_index.md#2-get-the-target-for-your-project).
+The DNS record for this target URL automatically resolves to the IP addresses of the routers in your project's region.
+When a router's IP address changes,
+the DNS record is automatically and immediately updated.
+This prevents downtime without intervention from you.
+ 
+Note that `CNAME` records can't point to [apex domains](../../other/glossary.md#apex-domain).
+Check with your DNS provider if they offer [workarounds](#workarounds-for-apex-domains).
+ 
+## Workarounds for apex domains
+
+Although `CNAME` records can't point to apex domains,
+most DNS providers offer workarounds.
+Contact your DNS provider and choose the option that works for you.
+
+Note that workarounds are also available for use with a CDN.
+For example, [Fastly](../cdn/fastly.md) offers [Anycast options](https://docs.fastly.com/en/guides/using-fastly-with-apex-domains) 
+and [Cloudflare](../cdn/cloudflare.md) offers [`CNAME` flattening options](https://developers.cloudflare.com/dns/additional-options/cname-flattening/).
```

Comment:
:hammer_and_wrench: :arrow_forward: With #2746, I'd be in favor of keeping the exact explanation of what the workaround is in one place. I know it's unlikely they'll change the name or anything, so maybe I shouldn't worry so much about repeating content. But I'd probably go with something like this:

```suggestion
If you use a CDN, you likely already have a workaround in place,
such as with [Fastly](../cdn/fastly.md#3-handle-apex-domains)
and [Cloudflare](../cdn/cloudflare.md#3-handle-apex-domains).
```

Not a blocking idea though.